examples_test.go

```diff
@@ -0,0 +1,110 @@
+package jsonschema_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/invopop/jsonschema"
+)
+
+type SampleUser struct {
+	ID          int                    `json:"id"`
```

Comment:
Should we use int64 or int32, for consistency across 32- and 64-bit systems?

Reply:
This is just an example, so I'm not too worried.